bucket/boxes.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "A command line filter program that draws ASCII art boxes around your input text",
+    "version": "1.2",
+    "license": "GPLv2",
```

Comment:
Should be `GPL-2.0-or-later` (See license identifiers on https://spdx.org/licenses/)